### ISSUE TITLE
Tormenting the Softknuckles not working?

### ISSUE THREAD

**Issue by nickagnone** [dalaranwow/dalaran-wow#5974]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

"Tormenting the Softknuckles" quest in Sholazar Basin asks you to poke a couple friendly NPC's with an item and a boss will spawn. I have been poking hundreds of Monkeys and no boss has spawned.

**Expected behaviour**: Tell us what should happen instead
Poking numerous monkeys results in boss being spawned

**Steps to reproduce the problem**:

1. Accept "Tormenting the Softknuckles" quest
2. Poke Monkeys with quest item
3. No boss spawns

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by MalkaviaDW** [comment]
Hello there, 

The quest works, keep using the item until the NPC appears, it will do.

Cheers!
